tools/tests/test_sync_equity_bars_data.py

```
@@ -0,0 +1,84 @@
+
+    target_date = datetime(2025, 6, 1, tzinfo=UTC)
+
+    with patch(
+        "tools.sync_equity_bars_data.requests.post", return_value=mock_response
+    ) as mock_post:
+        status_code, response_text = sync_equity_bars_for_date(
+            base_url="http://localhost:8080",
+            date=target_date,
+        )
+
+    assert status_code == 200  # noqa: PLR2004
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `PLR2004`)

Remove unused `noqa` directive

(RUF100)